CONTRIBUTING.md

```diff
@@ -54,9 +51,10 @@ That includes descriptive commit messages.
 5.	Please keep your PRs as small as possible, _i.e._ if you plan to perform a huge change, do not submit a single and large PR for it.
 For an enhancement or larger feature, you can create a GitHub issue first to discuss.
 6.	Before you push, run these commands in your terminal to check if there are any errors:
-    * `npm run compile`: compiles typescript into javascript
-    * `npm run lint`: runs linting on the code
-    * `npm run validate-user-code`: Checks the generated type decleration files
-    * `npm run test`: runs unit and integration tests against the generated javascript code
+    * `npm install`: Installs dependencies
+    * `npm run compile`: Compiles TypeScript into JavaScript, checking for any errors along the way
+    * `npm run lint`: Runs linting to find stylistic/structural errors in the code
+    * `npm run validate-user-code`: Checks the validity of generated TypeScript type declaration files
+    * `npm run test`: Runs unit and integration tests against the generated javascript code(the generated code is in lib/ folder)
```

Comment:
`javascript` -> `JavaScript`

Reply:
We can also put `lib/` in between backticks

Reply:
👍🏻